acceptance-tests/Gemfile.lock

```diff
@@ -56,9 +72,12 @@ PLATFORMS
 
 DEPENDENCIES
   capybara
+  httparty
+  mail
+  pdf-reader
+  pry
   rspec
   selenium-webdriver
-  sinatra
 
 RUBY VERSION
    ruby 2.6.4p104
```

Comment:
Should there be a modified Gemfile in this PR?

Reply:
this is the lockfile not the Gemfile. that likly meens the lock change was not committed while back so its just

Reply:
this is the lockfile `sinatra` was removed from the gems a while back.